types/google-apps-script/apis/calendar/v3.d.ts

```diff
@@ -0,0 +1,588 @@
+// Type definitions for Google Apps Script 2019-03-22
+// Project: https://developers.google.com/apps-script/
+// Definitions by: mtgto <https://github.com/mtgto/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace GoogleAppsScript {
+  namespace Calendar_v3 {
```

Comment:
We don't need all these namespaces.
Just GoogleAppsScript.Calendar_v3

See this example:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/google-apps-script/apis/slides/v1.d.ts#L6

Reply:
Fix it.

https://github.com/mtgto/dts-google-apps-script-advanced/commit/ec58737259a833967b6b874132c92b46aeb5bbe3#diff-ed009b6b86b017532ef0489c77de5100R103